setup.py

```diff
@@ -10,6 +10,6 @@
 
 setup(description="pymosaic-fits",
       long_description=open('README.md').read(),
-      version='0.4.0',
+      version='0.4.1',
```

Comment:
rather please do not update the version like that, it will introduce confusion with what I update. there should be only one source of version increment